hapistrano.cabal

```diff
@@ -51,6 +51,7 @@ executable hap
   build-depends:       base
                      , hapistrano
                      , base-compat
+                     , optparse-applicative >= 0.13 && < 0.14
```

Comment:
@CristhianMotoche Does this work with optparse-applicative 0.12 (the one in the most recent LTS snapshot)? We could also test with 0.11 (the one in LTS 3.5, which is the one specified in the stack.yaml file) (and we can set the resolver to something more recent). If they all work, we can change this constraint to something like >= 0.11 && < 0.14 and remove the extra-deps section from the stack.yaml file.

Other than that, this looks good. Let me know if it's ready and we can accept this. I haven't used optparse-applicative, so I'll check the documentation and review again if it's ready.